core/src/main/java/greencity/mapping/AddEcoNewsDtoResponseMapper.java

```diff
@@ -20,21 +17,13 @@
  */
 @Component
 public class AddEcoNewsDtoResponseMapper extends AbstractConverter<EcoNews, AddEcoNewsDtoResponse> {
-    private LanguageService languageService;
-    private EcoNewsTranslationRepo ecoNewsTranslationRepo;
     private EcoNewsAuthorDtoMapper ecoNewsAuthorDtoMapper;
```

Comment:
No other classes should be here. Probbly this method of converter should help - convert(MappingContext<S,D> context)